src/schema/v2/filterArtworksConnection.ts

```diff
@@ -273,12 +273,27 @@ export const FilterArtworksFields = () => {
       type: new GraphQLList(Artist.type),
       description:
         "Returns a list of merchandisable artists sorted by merch score.",
-      resolve: ({ aggregations }, _options, { artistsLoader }) => {
+      args: {
+        size: {
+          type: GraphQLInt,
+          description: "The number of artists to return",
+        },
+      },
+      resolve: (
+        { aggregations },
+        { size }: { size?: number },
+        { artistsLoader }
+      ) => {
         if (!isExisty(aggregations.merchandisable_artists)) {
           return null
         }
+
+        const artistIdsToReturn = size
```

Comment:
#minor looks like someone passing in a `size: 0` could wind up loading all the artists and DOS'ing (altho if thats what happens by default, w/o including the arg at all, i guess that doesn't matter 🤔 )